### ISSUE TITLE
Get affected rows count on insert

### ISSUE THREAD

**Issue by mnesarco** [tekartik/sqflite#402]
How to know if the row was inserted in the following scenario:

1. TABLE_INVITATIONS's primary key type is TEXT, there is no auto_increment column defined in the table;

2. I want o insert a record but ignore any conflict.

3. I Need to know if the record was inserted or ignored.

```
var k = await db.insert(sql.TABLE_INVITATIONS, record, conflictAlgorithm: ConflictAlgorithm.ignore);
```

k is null (because of [1] I suppose)

Is there any mechanism to get affected rows count?

**Comment by alextekartik** [comment]
Even without auto_increment, SQLite has a ROWID returned on success.

I wrote a simple test:
https://github.com/tekartik/sqflite/blob/49f5f164b2eb5bb647aae4b0e2f72cc552dbcdf9/sqflite_common_test/lib/raw_test.dart#L697-L715

that I tested on Android/iOS/MacOS and using ffi (there was actually a bug here where the last inserted id was returned).

I would think that basically it is safe to assume that null will be returned on conflict.
